inst/qml/GeneralizedLinearModel.qml

```diff
@@ -0,0 +1,469 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+import QtQuick			2.8
+import QtQuick.Layouts  1.3
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP				1.0
+
+// All Analysis forms must be built with the From QML item
+Form {
+    id: form
+    function updateWeightsTitle() {
+        if (family.currentText == "Binomial")
+        {title = qsTr("Total Number of Trials")}
+        else
+        {title = qsTr("Weights")}
+        return title
+    }
+
+    VariablesForm
```

Comment:
Since a lot of this is shared with generalized mixed models, it would make sense to create a custom qml component. The question is whether this component should live inside of `jaspRegression` or `jaspMixedModels`... I think we could do it here and later reuse the component in the mixed models. Could you move this into a separate qml file? Some info about it is here: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-qml-guide.md#advanced-usage. Otherwise, let me know if you need advice how to do this